tests/core/test_core.py

```diff
@@ -580,10 +580,30 @@ def mock_typing() -> None:
         if active_conversation_info:
             set_footer_text.assert_has_calls(
                 [
-                    mocker.call([("footer_contrast", " hamlet "), " is typing"]),
-                    mocker.call([("footer_contrast", " hamlet "), " is typing."]),
-                    mocker.call([("footer_contrast", " hamlet "), " is typing.."]),
-                    mocker.call([("footer_contrast", " hamlet "), " is typing..."]),
+                    mocker.call(
+                        [
+                            ("footer_contrast", "hamlet "),
+                            ("footer", " is typing"),
+                        ]
+                    ),
```

Comment:
I'm a bit confused why this reformatting was necessary? If you drop the comma after the second list element, does black shrink it back again?